docs/src/tutorials/writing-a-userstyle.md

```diff
@@ -238,8 +226,8 @@ Combining all of the previous steps, we have a working userstyle!
 // vim:ft=less
 ```
 
-| Original                                | Themed                                |
-| --------------------------------------- | ------------------------------------- |
-| ![](./images/example.org-original.webp) | ![](./images/example.org-themed.webp) |
+| Original                                 | Themed                                 |
+| ---------------------------------------- | -------------------------------------- |
+| ![](../images/example.org-original.webp) | ![](../images/example.org-themed.webp) |
```

Comment:
This works on GitHub but not on the mdBook. Will most likely have to convert the entire table into HTML.

Reply:
No it's because of Git LFS I believe.

Reply:
Ah yeah you're right, forgot I was testing this on different machine.